osu.Game.Rulesets.Catch/Skinning/Argon/ArgonCatcher.cs

```diff
@@ -81,5 +83,11 @@ private void load()
                 },
             };
         }
+
+        protected override void Update()
```

Comment:
Surely this can be avoided by adjusting the visual elements to use centre origin...

Reply:
I didn't look too much, you might have better clues on how this one can be adjusted than I do.